frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -112,13 +154,55 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                                         type="primary"
                                         data-attr="launch-survey"
                                         onClick={() => {
-                                            launchSurvey()
+                                            LemonDialog.open({
+                                                title: 'Launch this survey?',
+                                                content: (
+                                                    <div className="text-sm text-muted">
+                                                        The survey will immediately start displaying to users matching
+                                                        the targeting conditions.
```

Comment:
```suggestion
                                                        the display conditions.
```

..since we use "Display conditions summary" at the end of the form. I'd also suggest removing the "Targeting" tab to "Display conditions"